packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -101,29 +101,32 @@ async function generateSideloadFile(app: OfficeApp, manifest: ManifestInfo, docu
  * to register an Office add-in in Office Online.
  * @param manifestPath Path to the manifest file for the Office Add-in.
  * @param documentUrl Office Online document url
- * @param devServerPort Port number for the dev-server.
  * @param isTest Indicates whether to append test query param to suppress Office Online dialogs.
  * @returns Document url with query params appended.
  */
-export async function generateSideloadUrl(manifestPath: string, documentUrl: string, devServerPort: number, isTest: boolean = false): Promise<string> {
+export async function generateSideloadUrl(manifestPath: string, documentUrl: string,  isTest: boolean = false): Promise<string> {
```

Comment:
`generateSideloadFile()` takes the `manifest: ManifestInfo` parameter. It would be better to do the same here, since it could already be read and just pass in the info.